guides/common/modules/proc_deploying-puppet-within-project.adoc

```diff
@@ -0,0 +1,28 @@
+[id="deploying-puppet-within-project_{context}"]
+= Deploying Puppet Within {Project}
+
+Before you can get started using Puppet in your {Project}, you have to
+ifdef::katello,satellite,orcharhino[]
+enable the Puppet integration and
+endif::[]
+deploy the Puppet agent on hosts.
+
+.Procedure
+ifdef::katello,satellite,orcharhino[]
+. Enable Puppet integration in your {Project}.
+For more information, see xref:Enabling_Puppet_Integration_{context}[].
+endif::[]
+. Sync the Puppet agent from the Puppet platform repository.
+For more information, see https://puppet.com/docs/puppet/7/install_puppet.html#enable_the_puppet_platform_repository[Enable the Puppet platform repository] in the _Open Source Puppet 7_ documentation.
```

Comment:
not sure if I read the conditions correctly, however this is not the case for satellite, there we install the puppet automatically or we provide an installer command to install it, not custom repo is required

Reply:
I've found out that this module is probably redundant, but we need to fine tune the rest.